libs/utils/utils.js

```diff
@@ -392,7 +392,7 @@ export function decorateSVG(a) {
     : textContent.substring(textContent.indexOf('|') + 1).trim();
   const img = document.createElement('img');
   img.setAttribute('loading', 'lazy');
-  img.src = localizeLink(sanitizedTextContent);
+  img.src = sanitizedTextContent.replace(new URL(sanitizedTextContent).origin, '');
```

Comment:
This will throw an error if `sanitizedTextContent` is not a valid url

Reply:
As I test with "wrong url" for the value of sanitizedTextContent, it does throw by the `catch` than just JS error from various places where using decorateSVG(). 
I still added `?` for the case of undefined. If we want to put it in under try/catch I could still move it under the try below : (https://github.com/adobecom/milo/blob/38c1a4828991964e5dba6e301df139bb90206e4b/libs/utils/utils.js#L400-L413)
Which I initially did then changed due to Chris M. comment above.